packages/api/src/api/expoJudgingSession/id/teams/get.ts

```diff
@@ -0,0 +1,38 @@
+import { Request, Response } from 'express';
+import { ExpoJudgingSession } from '@hangar/database';
+import { logger } from '../../../../utils/logger';
+
+export const get = async (req: Request, res: Response) => {
+  // sone functionality
+  const {
+    judge,
+    entityManager: em,
+    params: { id: ejsId },
+  } = req;
+
+  try {
+    const ejs = await em.findOne(ExpoJudgingSession, { id: ejsId as string });
+
+    if (!ejs) {
+      res.sendStatus(404);
+    }
+
+    await em.populate(judge, ['expoJudgingSessionContexts']);
+
+    const validSession = judge.expoJudgingSessionContexts
+      .getItems()
+      .filter((ejsc) => ejsc.expoJudgingSession.id === ejsId);
```

Comment:
Use the find method to get the one and only context matching your ID; doing this makes the below a lot simpler
```suggestion
      .find((ejsc) => ejsc.expoJudgingSession.id === ejsId);
```